backend/EasyMeets.Core/EasyMeets.Core.WebAPI/Controllers/UploadImageController.cs

```diff
@@ -18,16 +17,23 @@ public UploadImageController(IUploadFileService uploadFileService)
         }
 
         [HttpPut]
-        public async Task<IActionResult> UploadImageAsync(ImageUploadDTO imageUploadDTO)
+        public async Task<IActionResult> UploadImageAsync()
         {
-            var imageUrl = await _uploadFileService.UploadFileBlobAsync(imageUploadDTO.FilePath, imageUploadDTO.FileName, imageUploadDTO.UserId);
-
-            if (imageUrl is null)
-            {
-                return NotFound();
+            var formCollection = await Request.ReadFormAsync();
+            var file = formCollection.Files.First();
+            var fileName = Path.GetFileName(file.FileName);
+            try
+            { 
+                var imageUrl = await _uploadFileService.UploadFileBlobAsync(file, fileName, 2); if (imageUrl is null)
+                {
+                    return NotFound();
+                } 
+                return Ok(new { imagePath = imageUrl });
             }
-
-            return Ok();
+            catch (Exception ex)
+            {
+                return BadRequest(ex);
+            } 
```

Comment:
Why you didn't add parameters to the controller method?

Reply:
This is FormCollection, so I decided to use Request.ReadFormAsync() to get file, so that this code is cleaner and for me looks better than using:
public async Task<IActionResult> UploadImageAsync(IFormCollection data)
        {
            string fname = data["fname_name"]; 
            ///...
        }
  And also as I understand form name should be known (["fname_name"]), I this it is better be not bind to form name 
  But I can use the second variant, if it is necessary.